opencti-platform/opencti-graphql/src/modules/internal/csvMapper/csvMapper-utils.ts

```diff
@@ -31,6 +31,12 @@ export interface CsvMapperSchemaAttributes {
   attributes: CsvMapperSchemaAttribute[]
 }
 
+// TS typeguard on CsvMapperRepresentation
+const isCsvMapperRepresentation = (object: any): object is CsvMapperRepresentation => {
+  // this is a basic validation; TODO: json validation
```

Comment:
We could do a JSON validation through ajv (we already have this lib in the project).

This is significant work though that should be done once-and-for-all and I'm unsure it's worth it right now.
No strong validation means we can make evolution easier on the JSON internal format.

Up to debate though